examples/sundials/ex26.cpp

```diff
@@ -0,0 +1,358 @@
+//                                MFEM Example 26
```

Comment:
Is there a chance that these will become a regular MFEM examples in the future?

If not, it will be better to move them to a new miniapps directory, maybe `miniapps/adjoint` and give them descriptive names following the format from the other miniapps.

Reply:
Alright. I think the parallel example is a sundials version of the advection diffusion problem but that also calculates adjoints. I can put it in `miniapps/adjoint` but use the same example number as mfem?

Reply:
Can you clarify which source files specifically you are referring too?

The idea of e.g examples/sundials/ex9.cpp is that it essentially examples/ex9.cpp with some SUNDIALS additions and the users can just diff the two files to see them. If that's not the case (and will not be the case in the near future) then the code should probably be a miniapp

Reply:
I'm referring to what is called `ex27p.cpp` (formerly `test_Advdiff.cpp`) at the moment.

I understand the idea behind the naming scheme. This is why I had initially named them differently, because they do not conform to idea behind the MFEM naming scheme ("ex" and corresponding to the example number). Previous reviewers suggested I change the name, but I think I was unclear with this point.

I will put what is currently `ex26.cpp` into `miniapps/adjoint`. My question is since `ex27p.cpp` is essentially a variation of `ex9` but with an adjoint calculation, would you want that to stay in the `example/sundials` directory or should I put it in `miniapps/adjoint`. Or should I call it `ex9.cpp` but put it in the `miniapps/adjoint` directory?

Reply:
It doesn't seem to me that the second example is version of Example 9. Maybe the equation solved is similar, but "version" here means that you can diff the two sources to learn how to use something.

I will move it to the `miniapps/adjoint` and call it  `advection-diffusion` or something